### PR TITLE
[rm] Remmina plugin

### PR DESCRIPTION
**Pull request by okb12**
Finally had the time to push it to here. Cheers.

### CODE REVIEW

**Comment by okb12** [comment]
Ok, turns out I don't get python... Can someone point me in the right direction for securing subprocess usage?

**Comment by bergercookie** [comment]
A couple of notes:

- I'll deal with Codacy, it's more of a PITA rather than useful tbh..
  I suppose `subprocess` could be linked with arbitrary code execution if not treated correctly..

- Could you squash into a single commit - also make sure you format the git message in imperative as described in this post: https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html

File: plugins/remmina/__init__.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import configparser
+
+__iid__ = "PythonInterface/v0.2"
+__prettyname__ = "Remmina"
+__version__ = "0.1"
+__trigger__ = "rm"
+__author__ = "Oğuzcan Küçükbayrak"
+__dependencies__ = ["remmina"]
+
+iconPath = "/usr/share/icons/hicolor/scalable/apps/org.remmina.Remmina.svg"
```
Comment (bergercookie) [inline plugins/remmina/__init__.py]: What if the icon is not found there? Could you perhaps copy the icon under the same directory and find that instead?
I'm doing the same for all the other plugins of this repo.

File: plugins/remmina/__init__.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+connectionsPath = "%s/.local/share/remmina" % (os.environ['HOME'])
+protoIconPath = "/usr/share/icons/hicolor/scalable/emblems/remmina-%s-symbolic.svg"
```
Comment (bergercookie) [inline plugins/remmina/__init__.py]: what's the difference between this and the `iconPath`?

**Review by bergercookie — CHANGES_REQUESTED** [review]

**Comment by bergercookie** [comment]
I think it would make more sense to change the default trigger to `rem` instead of `rm`.
I'm always thinking of the corresponding UNIX `rm` command when typing it :)

What do you think?

File: plugins/remmina/__init__.py [hunk 3]
```diff
@@ -23,12 +23,10 @@
...
 def runRemmina(cf=""):
     args = (['remmina'], ['remmina', '-c', cf])[len(cf) > 0]
-    subprocess.Popen(
-        args,
-        cwd=os.environ['HOME'],
-        stdout=subprocess.PIPE,
-        stderr=subprocess.STDOUT
-    )
+    try:
+        subprocess.check_output(args)
```
Comment (bergercookie) [inline plugins/remmina/__init__.py]: No need for the `try-catch` statement, I'll remove this codacy check anyway.

File: plugins/remmina/__init__.py (hunk 2 above)
Comment (okb12) [reply plugins/remmina/__init__.py]: `protoIconPath` is the icon of the connection's protocol. For example; `remmina-rdp-symbolic.svg` or `remmina-ssh-symbolic.svg`. I'll check the sizes and try to pack some of the icons with the module if they are not too large as requested with the above comment.

File: plugins/remmina/__init__.py (hunk 1 above)
Comment (okb12) [reply plugins/remmina/__init__.py]: This icon is somehow not the same as the one used by albert app launcher anyways, I'll pack the correct one with the module.

**Comment by okb12** [comment]
I agree with you about `rm` conflict. My personal preference is `rm` but it should be distinguishable for general usage.

**Comment by okb12** [comment]
Just for clarification; I do not claim any rights about the icons used in the extension. All icons belong to the [remmina project](https://remmina.org/).

**Comment by bergercookie** [comment]
Excellent!
Thanks for your contribution Oğuz!

**Pull request merged**